src/components/WearablePreview/WearablePreview.controller.ts

```diff
@@ -89,6 +141,10 @@ export function createController(id: string): IPreviewController {
     throw new Error(`Could not find an iframe with id="${id}"`)
   }
 
+  isControllerReady = false
+  pendingMessages.length = 0
```

Comment:
Same here, we should do:

```suggestion
  pendingMessages = []
```